roles/lib_openshift/tasks/main.yml

```diff
@@ -6,6 +6,8 @@
 
 - name: lib_openshift ensure python-ruamel-yaml package is on target
   package:
-    name: python2-ruamel-yaml
+    name: "python{{ ansible_python.version.major }}-ruamel-yaml"
     state: present
-  when: not ostree_booted.stat.exists
+  when:
+  - not containerized | default(false)
```

Comment:
Why test for containerized? Unless the remote system is atomic, we should go ahead and install dependencies that affect modules.